pkg/api-server/dataplane_overview_ws.go

```diff
@@ -80,8 +81,12 @@ func (r *overviewWs) inspectDataplanes(request *restful.Request, response *restf
 	}
 
 	tags := parseTags(request.QueryParameters("tag"))
+	gateWayFilterQueryParam := request.QueryParameter("gateway")
+	if gateWayFilterQueryParam == "true" {
+		gatewayFilterTag := map[string]string{"service": "gateway"}
+		overviews.RetainMatchingTags(gatewayFilterTag)
+	}
```

Comment:
Gateway dataplane does not imply that there will be a service tag = gateway.
Here is example from kuma.io docs
```
type: Dataplane
mesh: default
name: kong-01
networking:
  address: 10.0.0.1
  gateway:
    tags:
      service: kong
  outbound:
  - port: 33033
    service: backend
```
as you can see, gateway dataplane is a dataplane without inbound section and with gateway section. The service tag can be anything ex. kong